apps/web/src/components/layout/components/OrganizationSelect.cy.tsx

```diff
@@ -0,0 +1,58 @@
+import { QueryClient, QueryClientProvider } from 'react-query';
+import OrganizationSelect from './OrganizationSelect';
+import { TestWrapper } from '../../../testing';
+import { Select } from '../../../design-system';
+
+describe('Organization Select Component Test', () => {
+  // Get the locator
+  const organizationSelectComponentSelector = '[data-test-id="organization-switch"]';
+  const queryClient = new QueryClient();
+
+  // TEST 1
+  it('should Render the Organization Select Component', () => {
+    cy.mount(
+      <TestWrapper>
+        <QueryClientProvider client={queryClient}>
+          <OrganizationSelect />
+        </QueryClientProvider>
+      </TestWrapper>
+    );
+  });
+
+  // TEST 2
+  it('should get the organization-switch data test id', () => {
+    cy.mount(
+      <TestWrapper>
+        <QueryClientProvider client={queryClient}>
+          <OrganizationSelect />
+        </QueryClientProvider>
+      </TestWrapper>
+    );
```

Comment:
This test is missing the check. Like this has no difference with the render test.